apps/frontend/src/libs/components/sidebar/sidebar-item/helper/get-class-name-nav.helper.ts

```diff
@@ -0,0 +1,7 @@
+import styles from "../styles.module.css";
+
+const getClassName = ({ isActive }: { isActive: boolean }): string => {
```

Comment:
let's keep this helper inside SidebarItem component, it most likely will depend on other props later, and usually we keep className related logic inside components